system.go

```diff
@@ -667,6 +656,55 @@ func (av *AirVantage) ConfigureCommunication(hbState string, hbPeriod int, srSta
 	return string(res.Operation), nil
 }
 
+func (av *AirVantage) CreateDataset (name string, description string, configuration []string, appId string) (DataSet, error) {
+
+	var dataset DataSet
+	dataset.Name = name
+	dataset.Description = description
+	dataset.Configuration = configuration
+	dataset.Application = appId
+
+	js, err := json.Marshal(&dataset)
+	if err != nil {
+		return DataSet{
+			Uid:           "",
+			Name:          "",
+			Description:   "",
+			Configuration: nil,
+			Application:   "",
+		}, err
+	}
+
+	ccUrl := av.URL("/api/v2/datasets")
```

Comment:
you are requesting dataset V2 but the format you use is for V1

Reply:
Done